lib/src/options/data_class_options.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:data_class_plugin/data_class_plugin.dart';
+import 'package:data_class_plugin/src/extensions.dart';
+import 'package:data_class_plugin/src/options/data_class_options_config.dart';
+import 'package:glob/glob.dart';
+
+@DataClass()
+class DataClassOptions {
+  /// Shorthand constructor
+  const DataClassOptions({
+    this.optionsConfig = const <String, OptionConfig>{},
+  });
+
+  final Map<String, OptionConfig> optionsConfig;
+
+  bool? _hasGlobMatch(String option, String filePath) {
```

Comment:
This can be replaced froom the extensions one right?

Reply:
True, I totally missed that. 
Thanks 🙏🏻